bin/generate-phpstorm-stubs-map.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+declare(strict_types=1);
+
+use PhpParser\Node;
```

Comment:
This file should still live within a namespace

Reply:
Done.